lib/rails_multitenant.rb

```diff
@@ -6,4 +6,8 @@
 
 require "rails_multitenant/middleware/extensions"
 
+module RailsMultitenant
+  delegate :get, :[], :fetch, :set, :[]=, :delete, :with_isolated_registry, to: :GlobalContextRegistry
+end
```

Comment:
If we assumed ActiveSupport >= 5.1, we could use `delegate_missing_to` instead of this selection I chose. But even with that assumption, I'm ambivalent about increasing the surface area without any demand.

Within Salsify, a shorthand for `current tenant = 'some ID'` provides 95% of the value. Not sure if other users have similar  patterns, but it seems to fall out that way naturally for us.